### PR TITLE
Remove PrecompileWeb and .suo files

### PR DESCRIPTION
**Pull request by lavanyachandramouli**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
-/config-tool/SAMLBridgeConfiguration/SAMLBridgeConfiguration.suo
+*.suo
```
Comment (wiarlawd) [inline .gitignore]: You're going to want to add /PrecompiledWeb/ here as well, if this is where the solution will drop the files as before.

**Review by wiarlawd — CHANGES_REQUESTED** [review]
Be careful with the commit message. You can't start with two adjacent lines like that. It looks like GitHub (but not git) is just ignoring the second line. You must have a 58 character summary line, optionally followed by a blank line and an extended description.

John L

File: .gitignore (hunk 1 above)
Comment (lavanyachandramouli) [inline .gitignore]: Done.

**Review by wiarlawd — APPROVED** [review]
LGTM

**Pull request merged**